esterni/piano-di-qualifica/components/B-valutazioni-per-il-miglioramento.tex

```diff
@@ -81,5 +83,47 @@ \subsubsection{Errori nei documenti}
 Gli errori riguardanti le norme sono stati commessi soprattutto nelle prime fasi di stesura dei documenti e sono stati subito corretti nella prima parte di verifica per ogni documento quando tutto il gruppo ha appreso meglio le norme interne imposte.
 Per i periodi troppo verbosi è stato utile l'utilizzo dell'Indice di Gulpease, ma soprattutto una buona comunicazione all'interno del gruppo per rimediare a frasi troppo lunghe.
 
+\subsection{Secondo periodo (RP)}
+Vengono esposti i problemi che sono sorti nella seconda parte di lavoro valutando di conseguenza quali soluzioni adottare in merito.
```

Comment:
Vengono esposti i problemi che sono sorti nella seconda parte di lavoro, valutando di conseguenza quali soluzioni adottare in merito.